package.json

```diff
@@ -38,7 +39,7 @@
     "moment": "^2.22.0",
     "moment-timezone": "^0.5.14",
     "moving-average": "^1.0.0",
-    "naf-janus-adapter": "^0.5.2",
+    "naf-janus-adapter": "https://github.com/mozilla/naf-janus-adapter#feature/blocking",
```

Comment:
Before landing this PR we should actually just get `naf-janus-adapter` deployed to npm with the new changes and use a version number

Reply:
👍 Sounds good. I'll use yarn link in the meantime.  I changed it here so no one else had to set up the link to test it, but I'll change it back.